### ISSUE TITLE
[Urgent] Tests fail

### ISSUE THREAD

**Issue by shakedzy** [shakedzy/dython#139]
Tests related to `associations` have begun to fail. even though there was no code change..

**Comment by shakedzy** [comment]
I tend to believe it is a dependency issue. See [here](https://github.com/shakedzy/dython/actions/runs/3050783777/jobs/4918266837) last successful build, and [here](https://github.com/shakedzy/dython/actions/runs/3300261456/jobs/5444560329) current fail.

**Comment by shakedzy** [comment]
Indeed, Pandas related issue. All tests pass with `pandas=1.4.4`, several failures with `pandas==1.5.1`. Needs to deep dive into the cause and changes.

**Comment by shakedzy** [comment]
As a quick-fix, I'll limit `pandas` version to be below 1.5.0. This is a new version, and might have incompatibilities with other dependencies.

**Comment by enrir** [comment]
I can't found which Pandas change between ver. 1.4.4 and 1.5.0 is causing this issue, but the problem seems related to the dtypes of the `corr` internal variable in the `association` method. One possibile solution could be to explicitly declare the dtype [here](https://github.com/shakedzy/dython/blob/8367244275bc452faa5bdd5de58a2ac3d32855c7/dython/nominal.py#L580) when the empty DataFrame is initialized.
